.eslintrc.js

```diff
@@ -1,8 +1,5 @@
 module.exports = {
   extends: ['airbnb-typescript-prettier'],
-  parserOptions: {
-    project: './tsconfig.json',
-  },
```

Comment:
This was apparently not required. Its presence required having ESLint ignore every file outside the TS project (which preventing linting files like vite.config.js, .eslintrc.js, etc).